README-EN.md

```diff
@@ -54,9 +59,50 @@ dependencies {
 }
 ```
 
-## 2.2 Sync Project
+### 2.1.2 Sync Project
 Click "Sync Project with Gradle Files" button on the Android Studio's menu bar to download dependence files.
 
+
+## 2.2 Eclipse 
+### 2.2.1 Imports SDK jar
+Puts [zplayads.jar](./eclipseJar) into eclipse project's libs，android configure the build path. The main tips as fellows:
```

Comment:
Import [zplayads.jar](./eclipseJar) into the Eclipse project's libs, and configure the build path, the main steps as follows,